.github/workflows/hygiene.yml

```diff
@@ -0,0 +1,67 @@
+name: Hygiene
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+jobs:
+  hygiene:
+    name: Checks
+    runs-on: ubuntu-latest
+    steps:
+#      - name: GitHub Context
+#        run: echo $GITHUB_CONTEXT
+#        env:
+#          GITHUB_CONTEXT: ${{ toJson(github) }}
```

Comment:
Not sure what those commented-out lines are for.

Reply:
Oh, that can be done better. They display the `github` value for the action which makes up for the less than clear documentation for GitHub actions! However, after doing that, I discovered `success()` and `failure()` in the conditions, so I just pushed something which allows this to be code, but skipped by default (see https://github.com/dra27/ocaml/runs/1871830379?check_suite_focus=true)